README.md

```diff
@@ -9,7 +9,7 @@
     <br><br><br>
 </p>
 
-![jtd](https://user-images.githubusercontent.com/896475/47384541-89053c80-d6d5-11e8-98dc-dba16e192de9.gif)
+https://user-images.githubusercontent.com/85418632/211186159-7914f397-4b0b-4762-8a94-090bda4cf8ce.mp4
```

Comment:
```suggestion
![jtd](https://user-images.githubusercontent.com/85418632/211186159-7914f397-4b0b-4762-8a94-090bda4cf8ce.mp4) 
```

Reply:
If we're planning on adding alt text, let's make it slightly more descriptive - something like

```suggestion
![a video walkthrough of various Just the Docs features](https://user-images.githubusercontent.com/85418632/211186159-7914f397-4b0b-4762-8a94-090bda4cf8ce.mp4)
```